src/cascadia/TerminalApp/Profile.cpp

```diff
@@ -740,6 +762,125 @@ std::wstring Profile::EvaluateStartingDirectory(const std::wstring& directory)
     }
 }
 
+// Method Description:
+// - Helper function for converting a user-specified font weight value to its corresponding enum
+// Arguments:
+// - The value from the settings.json file
+// Return Value:
+// - The corresponding value which maps to the string provided by the user
+uint16_t Profile::_ParseFontWeight(const Json::Value& json)
+{
+    if (json.isUInt())
+    {
+        auto fontWeight = json.asUInt();
+
+        // If it's a valid fontWeight, pass it through. Otherwise, ignore it.
+        switch (fontWeight)
+        {
+        case 100:
+        case 200:
+        case 300:
+        case 350:
+        case 400:
+        case 500:
+        case 600:
+        case 700:
+        case 800:
+        case 900:
+        case 950:
+            return static_cast<uint16_t>(fontWeight);
+        }
+    }
+
+    if (json.isString())
+    {
+        auto fontWeight = json.asString();
+        if (fontWeight == FontWeightThin)
+        {
+            return 100;
+        }
+        else if (fontWeight == FontWeightExtraLight)
+        {
+            return 200;
+        }
+        else if (fontWeight == FontWeightLight)
+        {
+            return 300;
+        }
+        else if (fontWeight == FontWeightSemiLight)
+        {
+            return 350;
+        }
+        else if (fontWeight == FontWeightNormal)
+        {
+            return 400;
+        }
+        else if (fontWeight == FontWeightMedium)
+        {
+            return 500;
+        }
+        else if (fontWeight == FontWeightSemiBold)
+        {
+            return 600;
+        }
+        else if (fontWeight == FontWeightBold)
+        {
+            return 700;
+        }
+        else if (fontWeight == FontWeightExtraBold)
+        {
+            return 800;
+        }
+        else if (fontWeight == FontWeightBlack)
+        {
+            return 900;
+        }
+        else if (fontWeight == FontWeightExtraBlack)
+        {
+            return 950;
+        }
+    }
+
+    return DEFAULT_FONT_WEIGHT;
+}
+
+// Method Description:
+// - Helper function for converting a FontWeight to its corresponding string
+//   value.
+// Arguments:
+// - closeOnExitMode: The enum value to convert to a string.
+// Return Value:
+// - The string value for the given FontWeight
+std::string_view Profile::_SerializeFontWeight(const uint16_t fontWeight)

```

Comment:
If I accidentally left in the Serialize functions, it was a mistake. you can remove this one.